v20.2/cockroach-sql.md

```diff
@@ -146,6 +146,7 @@ Command | Usage
 `\h <statement>`<br>`\hf <function>` | View help for specific SQL statements or functions. See [SQL shell help](#help) for more details.
 `\l` | List all databases in the CockroachDB cluster. This command is equivalent to [`SHOW DATABASES`](show-databases.html).
 `\dt`<br>`d` | Show the tables of the current schema in the current database. These commands are equivalent to [`SHOW TABLES`](show-tables.html).
+`\dT` | <span class="version-tag">New in v20.2:</span> Show the [user-defined types](enum.html) in the current database. This command is equivalent to [`SHOW ENUMS`](show-enums.html) and [`SHOW TYPES`](show-types.html).
```

Comment:
you should keep that to just SHOW TYPES to not give the mistaken impression that \dT is specific to enums

Reply:
Done. TFTR!